src/core/ext/filters/client_channel/http_proxy_mapper.cc

```diff
@@ -77,12 +72,29 @@ bool ServerInCIDRRange(absl::string_view server_host,
   uint32_t mask_bits = 0;
   if (absl::SimpleAtoi(possible_cidr.second, &mask_bits)) {
     grpc_sockaddr_mask_bits(&*proxy_address, mask_bits);
-    return grpc_sockaddr_match_subnet(&*server_address, &*proxy_address,
+    return grpc_sockaddr_match_subnet(&server_address, &*proxy_address,
                                       mask_bits);
   }
   return false;
 }
 
+bool AddressIncluded(
```

Comment:
Please document, since the behavior of this function is not clear from its API.

Reply:
Tried refactoring the code to make it easier to follow.